roxie/ccd/ccdfile.cpp

```diff
@@ -633,14 +822,100 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
             return FileNotFound;
     }
 
-    ILazyFileIO *openFile(const char *lfn, unsigned partNo, const char *localLocation,
+#ifdef _CONTAINERIZED
```

Comment:
It would need more work (left as an exercise for a future ticket) to be useful in bare-metal. In particular without the shared dllserver filesystem this approach is not viable.

Bare-metal systems CAN use the control: queries to get/set cache info, to be stored in a location of the operator's choice...

Reply:
Worth creating a jira - if a different variable was used then a locally mounted directory that was shared could be configured for this purpose.

Reply:
https://track.hpccsystems.com/browse/HPCC-24737